doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc

```diff
@@ -102,3 +106,14 @@ You can open the Cockpit console for a virtual machine from the image:ssui-acces
 image:cockpit-login-nb.png[Cockpit Login]
 . Once you have logged in, you will see the tabs for the dashboard and the individual machines added to Cockpit. 
 image:cockpit-ui-nb.png[Cockpit UI]
+
+[[vm-power-operations]]
+=== Power Operating the Virtual Machine
```

Comment:
I think this would sound better as something like "Controlling virtual machine power operations". Thoughts?

Reply:
I agree; modified. PMs/Devs sometimes use that term but this sounds better- "Controlling virtual machine power states". Thanks.